test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -360,6 +373,13 @@ TEST_VM(os, jio_snprintf) {
 //  (from multiple calls to os::reserve_memory)
 static address reserve_multiple(int num_stripes, size_t stripe_len) {
   assert(is_aligned(stripe_len, os::vm_allocation_granularity()), "Sanity");
+
+#ifdef __APPLE__
+  // Workaround: try reserving memory with executable flag set to True
+  // to figure out if such operation is supported on this macOS version
```

Comment:
This comment really belongs on can_reserve_executable_memory now.

Reply:
I'd prefer to have a comment at the place of the workaround. How about changing it slightly to avoid implementation details like so:

```
  // Workaround: try reserving executable memory to figure out
  // if such operation is supported on this macOS version
```